src/typings.d.ts

```diff
@@ -1,4 +1,4 @@
-/* SystemJS module definition */
-declare var module: {
-	id: string;
-};
+// icon modules goes here for strict applications
+declare module '@carbon/icons/es/notification/16';
```

Comment:
There could be a catch-all module declaration for icons so they don't need to add one by one?